docs/tutorials/avro.ipynb

```diff
@@ -0,0 +1,379 @@
+{
```

Comment:
<p>Can we split the content in this section into:</p><ul><li>About dataset (where we cover the schema exploration)</li><li>Prepare tfio dataset (where the AvroRecordDataset is prepared, explain how we can use <code>num_parallel_reads</code>  and how it affects the io performance.)</li><li>Train tf.keras models (Explain how we can train using the AvroRecordDataset)</li><li>Inference (Explain how we can infer using the AvroRecordDataset)</li></ul><p> </p><p>Refer: https://www.tensorflow.org/io/tutorials/elasticsearch</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/io/pull/1250/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>